section1/exercises/booleans.rb

```diff
@@ -10,7 +10,19 @@
 p 7 > 2
 
 # YOU DO: print to the terminal the result of "hello" is equal to "Hello":
-
+#p Hello = hello
+#irb(main):005:0> p Hello == hello
+p "Traceback (most recent call last):
+        5: from /Users/wyatt/.rbenv/versions/2.7.2/bin/irb:23:in `<main>'
+        4: from /Users/wyatt/.rbenv/versions/2.7.2/bin/irb:23:in `load'
+        3: from /Users/wyatt/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
+        2: from (irb):4
+        1: from (irb):5:in `rescue in irb_binding'
+NameError (uninitialized constant Hello)"
+#I'm not sure if this was the intent of the exercise but I do know that the reasoning
```

Comment:
Yes and no!

The output you got and put in this doc above is actually an error, because you were comparing `Hello` to `hello` - as opposed to `"Hello"` and `"hello"` - if you compare those, you would get `false`, but not an error. It's those quotation marks that make the difference. 

The error you got is Ruby saying "I don't know a keyword called `Hello` so I'm freaking out and going to stop"